api/managementpb/backup/backups.proto

```diff
@@ -206,12 +206,14 @@ message RemoveScheduledBackupResponse {}
 
 message GetLogsRequest {
   string artifact_id = 1 [
-    (validator.field) = {
-      string_not_empty: true
-    }
+    deprecated = true
   ];
   uint32 offset = 2;
   uint32 limit = 3;
+  oneof job_id {
+    string backup_artifact_id = 4;
+    string restore_id = 5;
+  };
```

Comment:
That's really confusing. JobID is job id, how we can put there completely different IDs?
Also this breaks backward compatibility.

Reply:
I've changed it to `log_job_id` - does that help?

Re backward compatibility, we only deprecated the original `artifact_id`, so old clients should still work fine (but they won't be able to get restore logs, only backup logs).

Reply:
> I've changed it to log_job_id - does that help?

Not really, this is container for two ids(artifact_id and restore_id), it has nothing to do with jobs. We don't expose job concept on API level at all yet. Once it will happen then we will add `job_id` to that endpoint and all other IDs will be deprecated.

For now I think we can just add plain `string restore_id = 4`, that should be good enough for now.